### PR TITLE
Or 1921 set deploy config thanos sepolia

### PR DESCRIPTION
**Pull request by AaronLee22**
**Set-deploy-config-thanos-sepolia for FPS**

"faultGameAbsolutePrestate": "0x03ab262ce124af0d5d328e09bf886a2b272fe960138115ad8b94fdc3034e3155",
  "faultGameMaxDepth": 73,
  "faultGameClockExtension": 10800,
  "faultGameMaxClockDuration": 302400,
  "faultGameGenesisBlock": ,
  "faultGameGenesisOutputRoot": "",
  "faultGameSplitDepth": 30,
  "faultGameWithdrawalDelay": 604800,
  "preimageOracleMinProposalSize": 126000,
  "preimageOracleChallengePeriod": 86400,
  "proofMaturityDelaySeconds": 604800,
  "disputeGameFinalityDelaySeconds": 302400,
  "respectedGameType": 0,
  "useFaultProofs": true,

**Have to set before the upgrade**

faultGameGenesisBlock: The block number at the time of applying FPS, before the upgrade.
faultGameGenesisOutputRoot: The hash of the block  at the time of applying FPS,  before the upgrade.

### CODE REVIEW

File: packages/tokamak/contracts-bedrock/deploy-config/thanos-sepolia.json [hunk 1]
```diff
@@ -46,9 +49,20 @@
...
   "requiredProtocolVersion": "0x0000000000000000000000000000000000000003000000010000000000000000",
   "recommendedProtocolVersion": "0x0000000000000000000000000000000000000003000000010000000000000000",
-  "nativeTokenName": "Tokamak Network Token",
-  "nativeTokenSymbol": "TON",
-  "nativeTokenAddress": "0xa30fe40285b8f5c0457dbc3b7c8a280373c40044",
+  "faultGameAbsolutePrestate": "0x03ab262ce124af0d5d328e09bf886a2b272fe960138115ad8b94fdc3034e3155",
+  "faultGameMaxDepth": 73,
+  "faultGameClockExtension": 10800,
+  "faultGameMaxClockDuration": 302400,
+  "faultGameGenesisBlock": ,
```
Comment (steven94kr) [inline packages/tokamak/contracts-bedrock/deploy-config/thanos-sepolia.json:56]: I think we need double quota if there is no value!

**Review by theo-learner — COMMENTED** [review]
I have checked that all packages is well built. But to be clear, can you define the range of the review? Please let us test method or something if you have it.

File: packages/tokamak/contracts-bedrock/deploy-config/thanos-sepolia.json (hunk 1 above)
Comment (AaronLee22) [reply packages/tokamak/contracts-bedrock/deploy-config/thanos-sepolia.json:56]: You are right! I'll update the empty values.

**Comment by theo-learner** [comment]
How about making the page for the fault proof deploy params?

**Comment by AaronLee22** [comment]
> How about making the page for the fault proof deploy params?

Thank you for the suggestion! I’ll create a dedicated page for the Fault Proof deploy parameters,

**Comment by steven94kr** [comment]
> How about making the page for the fault proof deploy params?

https://docs.optimism.io/builders/chain-operators/configuration/rollup#fault-proofs

**Comment by theo-learner** [comment]
> > How about making the page for the fault proof deploy params?
> 
> https://docs.optimism.io/builders/chain-operators/configuration/rollup#fault-proofs

Sorry I missed it. Thanks @steven94kr . 
@AaronLee22 It's better to make the page if you have some additional information.

**Review by steven94kr — APPROVED** [review]
I re-run actions. It is weird... I think we can resolve the action later.
Thank you very much!

**Pull request merged**